demos/browser/app/meeting/CloudWatchLogger.ts

```diff
@@ -0,0 +1,86 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import LogLevel from '../../../../src/logger/LogLevel';
+import {ConsoleLogger} from '../../../../src/index';
+
+export default class CloudWatchLogger extends ConsoleLogger {
+  name: string;
+  level: LogLevel;
+  logCapture: string[] = [];
+  meetingID: string;
+  attendeeID: string;
+  logSeqNo: any;
```

Comment:
Please do not use abbreviations. e.g., => ```logSequenceNumber```.
(Sorry, I remember I've seen this convention somewhere, but I can't find that doc.)